### DISCUSSION TITLE
Enable `.modify skill` on the PTR.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7123] [Suggestions]
don't ask me for a description this needs no description leave me alone

**Comment by rakrak1775** [comment]
honey fix wth

**Comment by BeefSteaker** [comment]
There's .setskill , or does this do something different?

**Comment by Plan414** [comment]
My notes to learn skills 

![image](https://github.com/chromiecraft/chromiecraft/assets/62799885/344b7ad9-23bb-4d20-9fd0-b9e16ed1a248)
